### PR TITLE
Translate mysql_xdevapi `Result`

### PR DESCRIPTION
**Pull request by Fan2Shrek**

### CODE REVIEW

File: reference/mysql_xdevapi/mysql_xdevapi/result/getgeneratedids.xml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <modifier>public</modifier> <type>array</type><methodname>mysql_xdevapi\Result::getGeneratedIds</methodname>
+   <void />
+  </methodsynopsis>
+  <para>
+   Récupèere les valeurs _id générées par la dernière opération.
```
Comment (devnexen) [inline reference/mysql_xdevapi/mysql_xdevapi/result/getgeneratedids.xml:17]: Typo `Récupère`

Comment (devnexen) [reply reference/mysql_xdevapi/mysql_xdevapi/result/getgeneratedids.xml:17]: Quick question: what editor do you use ? if might be usefule using one which can highlight bad french wordings. If there is any.

Comment (Fan2Shrek) [reply reference/mysql_xdevapi/mysql_xdevapi/result/getgeneratedids.xml]: I mainly use VSCode, but spell checker extensions don't seem to work with XML files. :(

Comment (Fan2Shrek) [reply reference/mysql_xdevapi/mysql_xdevapi/result/getgeneratedids.xml]: I tried to make a better configuration, should be better now ;)

**Review by devnexen — APPROVED** [review]

**Pull request merged**